packages/pipeline-editor/src/canvas.ts

```diff
@@ -19,23 +19,31 @@ import { IconUtil } from '@elyra/ui-components';
 import { PathExt } from '@jupyterlab/coreutils';
 import { DocumentRegistry } from '@jupyterlab/docregistry';
 import { Contents } from '@jupyterlab/services';
-import { LabIcon, notebookIcon, pythonIcon } from '@jupyterlab/ui-components';
+import {
+  LabIcon,
+  notebookIcon,
+  pythonIcon,
+  rKernelIcon
+} from '@jupyterlab/ui-components';
 
 import { PipelineService } from './PipelineService';
 
 enum ContentType {
   notebook = 'notebook',
   python = 'python',
+  r = 'r',
```

Comment:
I think it may be better to call this `r_script` or `rScript` instead of having a one char name